presto-parser/src/main/java/com/facebook/presto/sql/parser/AstBuilder.java

```diff
@@ -1242,6 +1242,14 @@ public Node visitFunctionCall(SqlBaseParser.FunctionCallContext context)
         Optional<Expression> filter = visitIfPresent(context.filter(), Expression.class);
         Optional<Window> window = visitIfPresent(context.over(), Window.class);
 
+        Optional<OrderBy> orderBy;
+        if (context.ORDER() != null) {
+            orderBy = Optional.of(new OrderBy(visit(context.sortItem(), SortItem.class)));
+        }
+        else {
```

Comment:
Sorry, could you please elaborate? I'm not sure which part of the formatting is wrong.